### PR TITLE
abort on missing files, add cmeps

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Issues #12 , #10:

- adds abort if DATM input file is missing. Abort message will be written to PET logs and default log file. This feature was tested by attempting to integrate in time past the available data input files. The model aborted as expected.

- adds cpl_scalars for use with CMEPS

- simplifies the setting of the run clock

- adds dbug_flag read from nems.configure to control message printing. Setting dbug_flag = 0 will suppress message printing. Setting dbug_flag > 5 will print all messages.

- removes additional importstate instances where possible

Verification that the DATM clock is working and advancing as expected is shown here. Black dots are the export values over 24 hours from initial time 2011-10-01-00 at a single location on the global grid. Values exported from the DATM at each timestep are show as black dots; the blue dots are the values in the 6-hourly forcing files. Only CFSR is shown.

cfsr

### CODE REVIEW

**Comment by DeniseWorthen** [comment]
All Regression tests pass on Orion and Hera. Logs posted. 

[Hera](https://github.com/DeniseWorthen/DATM-MOM6-CICE5/blob/feature/updateDATM/tests/RegressionTests_hera.intel.log)
[Orion](https://github.com/DeniseWorthen/DATM-MOM6-CICE5/blob/feature/updateDATM/tests/RegressionTests_hera.intel.log)

File: DATM/AtmForce.F90 [hunk 1]
```diff
@@ -76,35 +86,48 @@ subroutine AtmForce(gcomp,exportState,externalClock,initmode,rc)
...
+      call ESMF_FieldBundleGet(AtmBundleFwd, &
+                               fieldName=trim(AtmBundleFields(ii)%field_name)//'_fwd', &
+                               field=field, rc=rc)
+      if (ChkErr(rc,__LINE__,u_FILE_u)) return
+
+      call ESMF_FieldRead(field, &
+                          fileName=trim(filename), &
+                          variableName = trim(varname), &
+                          rc=rc)
+      if (ChkErr(rc,__LINE__,u_FILE_u)) return
```
Comment (junwang-noaa) [inline DATM/AtmForce.F90:124]: I'd suggest to add following for the error in this ESMF_FieldRead, if there is any issue when reading the file, the code should abort, currently it only return to the calling routine.

call ESMF_LogWrite('Missing input fields in datm_data_table', ESMF_LOGMSG_INFO)
call ESMF_Finalize(endflag=ESMF_END_ABORT)

Comment (DeniseWorthen) [reply DATM/AtmForce.F90:124]: Yes, I will also add an abort if it tries to read the file but gets a return error.

**Comment by DeniseWorthen** [comment]
I just realized I hadn't merged my cpl_scalars branch (needed for CMEPS) after all. I will do that and re-run the regression tests.

**Comment by DeniseWorthen** [comment]
All Regression tests pass on Orion and Hera. Logs posted.

[Hera](https://github.com/DeniseWorthen/DATM-MOM6-CICE5/blob/feature/updateDATM/tests/RegressionTests_hera.intel.log)
[Orion](https://github.com/DeniseWorthen/DATM-MOM6-CICE5/blob/feature/updateDATM/tests/RegressionTests_orion.intel.log)

**Comment by DeniseWorthen** [comment]
@binli2337, @flampouris : This PR is ready now. Could you review? Thanks.

**Review by junwang-noaa — APPROVED** [review]

**Review by binli2337 — APPROVED** [review]

**Review by flampouris — APPROVED** [review]
Thank you @DeniseWorthen !

**Pull request merged**